common/beerocks/bcl/CMakeLists.txt

```diff
@@ -17,13 +19,16 @@ add_library(${PROJECT_NAME} ${bcl_sources})
 set_target_properties(${PROJECT_NAME} PROPERTIES VERSION ${prplmesh_VERSION} SOVERSION ${prplmesh_VERSION_MAJOR})
 
 # Includes
+include_directories(${PLATFORM_INCLUDE_DIR}/libnl3)
+include_directories(${PLATFORM_INCLUDE_DIR}/libnl-route3)
```

Comment:
Are you sure this is required?
The include directories should be automatically added by the `target_link_libraries` function.

Reply:
I tried to remove it, and it compiles. So apparently it is not needed.
Removing it.